/**:
  ros__parameters:
    port: "/dev/ttyUSB1"
    baud: 115200
    left_motor_id: 1
    right_motor_id: 2
    wheel_radius: 0.05
    wheel_tread: 0.38
    frame_id: "odom"
    timeout: 1.0

